environments/autostart.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Autostart"
+description: Learn how to configure automated environment rebuilds.
+---
+
+Coder automatically [turns off](../admin/environment-management/shutdown.md)
+unused environments to help you save on cloud usage costs. Turning off
+environments, however, requires you to turn them on manually the next day.
+Autostart can help eliminate this manual process.
+
+Autostart also allows you to configure automated environment rebuilds at a time
+that best suits your workflow. You can now expect your environments to be ready
+for you at the start of each workday.
+
+## Criteria for Autostart
+
+Your environment must meet the following criteria for you to use autostart:
+
+1. You've [set the autostart time and selected
+   environments](#enabling-autostart) to be started automatically at that time.
+
+1. Your environments are off at the autostart time (autostart won't work if your
+   environments are on to prevent the possibility of triggering a rebuild while
+   you're working).
+
+1. Your environment must be active. We define *inactive* environments as those
+   that haven't been opened in five days or more.
```

Comment:
All of these are numbered w/1. 

Is this intentional?

If these don't need to be numbered maybe we can just use bulllets or something

Reply:
Yeah, the 1s will render with proper numbering -- makes it so you can add steps without having to change everything by hand